charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.4/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Harvester Cloud Provider
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>= 2.6.1'
+  catalog.cattle.io/release-name: harvester-cloud-provider
+  catalog.cattle.io/ui-component: harvester-cloud-provider
+apiVersion: v2
+appVersion: v0.1.1
+description: A Helm chart for Harvester Cloud Provider
+maintainers:
+- name: harvester
+name: harvester-cloud-provider
+type: application
+version: 100.0.0+up0.1.4
```

Comment:
Needs keywords to make some of the UI search work. 'Infrastructure' makes sense.

Reply:
How do we add the keyword?

Reply:
https://helm.sh/docs/topics/charts/#the-chartyaml-file